style.css

```diff
@@ -0,0 +1,39 @@
+* {
+	box-sizing: border-box; 
+}
+
+html {
+	font-size: 20px; 
+}
+
+body {
+	background-color:#e3800f;
+    margin:0px;
+}
+
+header {
```

Comment:
![image](https://github.com/Joshsilas/project2/assets/137928354/4d9fac1c-3561-480e-856d-c617e663e8e2)
indentation is off (might be a github thing)